src/schemas/stoatConfigSchema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "required": ["version", "comment_template"],
```

Comment:
also should be updated in `stoatConfigSchemaRendered.json`

Reply:
Removed `comment_template` from the rendered type.

This field should probably be renamed to `comment_template_file` though.